skabase/SKAAlarmHandler/SKAAlarmHandler.py

```diff
@@ -11,18 +11,16 @@
 SKA Alarms and SKA/Element Alerts are rules-based configurable conditions that can be defined over multiple
 attribute values and quality factors, and are separate from the "built-in" TANGO attribute alarms.
 """
-
-# tango imports
-from tango import DebugIt
-from tango.server import run, DeviceMeta, attribute, command, device_property
-
-# Additional import
 # PROTECTED REGION ID(SKAAlarmHandler.additionnal_import) ENABLED START #
 #standard import
```

Comment:
Space after `#`.
`imports` rather than `import`.

Reply:
Fixed

Reply:
Checked and fixed in all other files.